Objects/dictobject.c

```diff
@@ -3826,7 +3831,7 @@ dictreviter_iternext(dictiterobject *di)
     PyObject *key, *value, *result;
 
     if (d->ma_values) {
-        if (i < 0) {
+        if (i < 0 || i >= d->ma_used) {
```

Comment:
Is this change still needed?

Reply:
@serhiy-storchaka 
No, it is not needed. I removed it on the latest commit.